doc/guides/west/install.rst

```diff
@@ -15,32 +15,13 @@ On Windows and macOS::
   pip3 install -U west
 
 .. note::
-   See :ref:`gs_python_deps` for additional clarfication on using the
+   See :ref:`gs_python_deps` for additional clarification on using the
    ``--user`` switch.
 
-The following are the typical locations where the west files will be
-installed if you followed the instructions above:
+Afterwards, you can run ``pip3 show -f west`` for information on where the west
+binary and related files were installed.
 
-.. tip::
-   You can use the ``pip3 show -f west`` command to display information about
-   the installed packaged, including file locations.
-
-* Linux:
-
-  * Executable: :file:`~/.local/bin/west`
-  * Package: :file:`~/.local/lib/python3.<x>/site-packages/west/`
-
-* macOS:
-
-  * Executable: :file:`/usr/local/bin/west`
-  * Package: :file:`/usr/local/lib/python3.<x>/site-packages/west/`
-
-* Windows:
-
-  * Executable: :file:`C:\\Python3<x>\\Scripts\\west.exe`
-  * Package: :file:`C:\\Python3<x>\\Lib\\site-packages\\west\\`
-
```

Comment:
Did you move these paths somewhere else in a different commit? Either way the commit message should say. The other changes are minor enough not to need a mention in the commit message, it's just this one.

Reply:
Sure, I'll update the commit message describing what happened here. This was a change that was requested in review of the original commit adding this file that didn't happen.